src/java/com/pb/mtctm2/abm/ctramp/ExplicitTelecommuteDMU.java

```diff
@@ -0,0 +1,256 @@
+package com.pb.mtctm2.abm.ctramp;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import org.apache.log4j.Logger;
+import com.pb.common.calculator.IndexValues;
+import com.pb.common.calculator.VariableTable;
+
+/**
+ * @author crf <br/>
+ *         Started: Apr 14, 2009 11:09:58 AM
+ */
+public class ExplicitTelecommuteDMU
+        implements Serializable, VariableTable
+{
+
+    protected transient Logger         logger = Logger.getLogger(ExplicitTelecommuteDMU.class);
+
+    protected HashMap<String, Integer> methodIndexMap;
+
+    private Household                  hh;
+    private Person                     person;
+    private IndexValues                dmuIndex;
+
+    private int                        mgraParkArea;
+    private int                        numFreeHours;
+    private int                        mstallsoth;
+    private int                        mstallssam;
+    private float                      mparkcost;
+    private int                        dstallsoth;
+    private int                        dstallssam;
+    private float                      dparkcost;
+    private int                        hstallsoth;
+    private int                        hstallssam;
+    private float                      hparkcost;
```

Comment:
What are all of the above variables? Are they from a parking model? Can we remove them all? (And the associated methods?